### PR TITLE
add a few more template vars

### PR DESCRIPTION
**Pull request by jstrachan**

### CODE REVIEW

**Comment by fabric8cd** [comment]
@jstrachan snapshot fabric8-tenant image is available for testing.  `docker pull docker.io/fabric8/fabric8-tenant:SNAPSHOT-PR-380-1`

**Comment by jstrachan** [comment]
@aslakknutsen these changes should be pretty innocent right? I wanna get these extra template vars in there ASAP so that we can start to use these (empty) expression in the Che YAML https://github.com/fabric8-services/fabric8-tenant-che/pull/33 and need fabric8-tenant to use empty values for these expressions in OSIO so that we can start to use these expressions for fabric8

**Comment by fabric8cd** [comment]
@jstrachan snapshot fabric8-tenant image is available for testing.  `docker pull docker.io/fabric8/fabric8-tenant:SNAPSHOT-PR-380-2`

**Comment by aslakknutsen** [comment]
@jstrachan Setting them like that won't make them empty tho?

**Comment by fabric8cd** [comment]
@jstrachan snapshot fabric8-tenant image is available for testing.  `docker pull docker.io/fabric8/fabric8-tenant:SNAPSHOT-PR-380-3`

File: configuration/configuration.go [hunk 1]
```diff
@@ -308,9 +310,11 @@ func (c *Data) GetTemplateValues() (map[string]string, error) {
 	}
 
 	return map[string]string{
-		"RECOMMENDER_EXTERNAL_NAME": c.v.GetString(varTemplateRecommenderExternalName),
-		"RECOMMENDER_API_TOKEN":     base64.StdEncoding.EncodeToString([]byte(c.v.GetString(varTemplateRecommenderAPIToken))),
-		"DOMAIN":                    c.v.GetString(varTemplateDomain),
+		"RECOMMENDER_EXTERNAL_NAME":      c.v.GetString(varTemplateRecommenderExternalName),
+		"RECOMMENDER_API_TOKEN":          base64.StdEncoding.EncodeToString([]byte(c.v.GetString(varTemplateRecommenderAPIToken))),
+		"CHE_KEYCLOAK_AUTH__SERVER__URL": c.v.GetString(varKeycloakAuthServerURL),
```
Comment (chmouel) [inline configuration/configuration.go]: is there a reason why this variable and the KEYCLOAK_CLIENT__ID have double underscores?

**Comment by jstrachan** [comment]
@aslakknutsen how should they bet set so that they are empty if not explicitly defined via env vars etc?

File: configuration/configuration.go (hunk 1 above)
Comment (jstrachan) [reply configuration/configuration.go]: those are the env vars that Che seems to want https://github.com/fabric8-services/fabric8-tenant-che/pull/33/commits/484191ea7df89d091fbaeec51073ef1899fd27f2

**Comment by jstrachan** [comment]
@aslakknutsen added code to use `""` if the config properties are not set - does that help?

**Comment by fabric8cd** [comment]
@jstrachan snapshot fabric8-tenant image is available for testing.  `docker pull docker.io/fabric8/fabric8-tenant:SNAPSHOT-PR-380-4`

**Pull request merged**